ui/components/Filters.js

```diff
@@ -409,6 +501,29 @@ function MesheryFilters({ updateProgress, enqueueSnackbar, closeSnackbar, user,
     );
   };
 
+  const handlePublish = (catalog_data) => {
+    updateProgress({ showProgress : true });
+    dataFetch(
+      `/api/filter/catalog/publish`,
+      { credentials : "include", method : "POST", body : JSON.stringify(catalog_data) },
+      () => {
+        updateProgress({ showProgress : false });
+        enqueueSnackbar("Filter Published!", {
```

Comment:
Why are you capitalizing "Published"? Please don't. 

Also, please don't yell at the user! It's not nice, is it?